sdks/go/pkg/beam/core/runtime/harness/harness.go

```diff
@@ -37,26 +38,15 @@ import (
 	"google.golang.org/protobuf/types/known/durationpb"
 )
 
-// StatusAddress is a type of status endpoint address as an optional argument to harness.Main().
-type StatusAddress string
-
 // TODO(herohde) 2/8/2017: for now, assume we stage a full binary (not a plugin).
 
 // Main is the main entrypoint for the Go harness. It runs at "runtime" -- not
 // "pipeline-construction time" -- on each worker. It is a FnAPI client and
 // ultimately responsible for correctly executing user code.
```

Comment:
Please add documentation around "expected" Environment variables. TBH, I don't mind the options approach to pass into main, but the  fetching form the Env vars would then happen in the init.go.